app/components/admin/widget/cards/row_component.rb

```diff
@@ -1,8 +1,10 @@
 class Admin::Widget::Cards::RowComponent < ApplicationComponent
-  attr_reader :card, :options
+  attr_reader :card, :header_section, :options
+  alias_method :header_section?, :header_section
 
-  def initialize(card, **options)
+  def initialize(card, header_section, **options)
```

Comment:
We're using an anonymous parameter here, but in the table component we're using a named parameter :thinking:. Is it on purpose? :thinking: